packs/feats/channel-rot.json

```diff
@@ -29,6 +29,15 @@
             "title": "Pathfinder Book of the Dead"
         },
         "rules": [
+            {
```

Comment:
This will stack the bonus damage on fist Strikes from both Accursed Touch and Channel Rot.

Additionally, consider removing the inline damage roll from the Notes, as it's already automated.

<img width="282" alt="Screen Shot 2025-01-19 at 10 21 07 AM" src="https://github.com/user-attachments/assets/8d59260e-8805-4c5d-b43f-00400a75b74f" />